### PR TITLE
Added heroku compablity 

### PR DESCRIPTION
**Pull request by Geektyper**

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -7,3 +8,4 @@ aiohttp[speedups]
 pillow
 hachoir
 requests
+selenium
```
Comment (r0ld3x) [inline requirements.txt:11]: No use of selenium now.

File: config.py [hunk 2]
```diff
@@ -10,5 +13,5 @@
 PASSWORD = ""  # redis password
 
 PRIVATE_CHAT_ID = -1001234567890  # CHAT WHERE YOU WANT TO STORE VIDEOS
-COOKIE = ""  # COOKIE FOR AUTHENTICATION (get from chrome dev tools) ex: "PANWEB=1; csrfToken=; lang=en; TSID=; __bid_n=; _ga=; __stripe_mid=; ndus=; browserid==; ndut_fmt=; _ga_06ZNKL8C2E=" (dont use this)
-ADMINS = [1317173146]
\ No newline at end of file
+COOKIE = os.getenv("COOKIE", None)  # COOKIE FOR AUTHENTICATION (get from chrome dev tools) ex: "PANWEB=1; csrfToken=; lang=en; TSID=; __bid_n=; _ga=; __stripe_mid=; ndus=; browserid==; ndut_fmt=; _ga_06ZNKL8C2E=" (dont use this)
```
Comment (r0ld3x) [inline config.py:16]: can you like this for all ?
like 

```
COOKIE = os.getenv("COOKIE", "") 
PRIVATE_CHAT_ID = os.getenv("PRIVATE_CHAT_ID ",-1001234567890)
```

Comment (Geektyper) [reply config.py:16]: I have deleted my fork can you provide me collab at repo ?

Comment (r0ld3x) [reply config.py:16]: > I have deleted my fork can you provide me collab at repo ?

yup wait

**Pull request closed without merging**